src/c++/perf_analyzer/perf_utils.h

```diff
@@ -145,4 +145,49 @@ template <Distribution distribution>
 std::function<std::chrono::nanoseconds(std::mt19937&)> ScheduleDistribution(
     const double request_rate);
 
+// Calls MPI_Init API.
+void MPI_Init(int* argc, char*** argv);
+
+// Calls MPI_Initialized API.
+bool MPI_Initialized();
+
+// Calls MPI_Comm_size API with MPI_COMM_WORLD communicator.
+int MPI_Comm_sizeWorld();
+
+// Calls MPI_Barrier API with MPI_COMM_WORLD communicator.
```

Comment:
Specifically, call out which of these functions can be called without checking for IsMPIRun.

Reply:
Added to comments 👍